components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -563,7 +621,9 @@ def getConfigValue(key):
         rectW = int((w+xOffset) * factor)
         rectH = int((h+yOffset) * factor)
         draw.rectangle((rectX, rectY, rectW, rectH), outline=lineColour)
-        draw.rectangle((rectX-1, rectY-1, rectW+1, rectH+1), outline=lineColour)    # hack to get line width of 2
+        draw.rectangle((rectX-1, rectY-1, rectW+1, rectH+1),
+                       outline=lineColour)
+        # hack to get line width of 2
```

Comment:
This comment should probably be on the line before the statement. Same goes for several similar changes in this file.